### ISSUE TITLE
Unknown key 'lr_scheduler_num_cycles'

### ISSUE THREAD

**Issue by 2Raven2** [sorryhyun/anima_lora#69]
I've given the newer versions a shot and they seem to be working pretty well. 

However, a hyperparameter called `lr_scheduler_num_cycles` is suddenly not valid, and so when `cosine_with_restarts` is used as the LR scheduler, it won't accept any value that I input for `lr_scheduler_num_cycles` and treats it like just like `cosine`. I'm confirming if this was removed at a later version, as I don't see this removal in your release notes for the last couple of versions.

This the error that shows up on the console:
```
  WARNING  [config]                             schema.py:293
                             D:\Generative-AI\anima_lora\output\d              
                             aemon\jobs\20260702-204948-1c9b30\co              
                             nfig.snapshot.toml:57: unknown key                
                             'lr_scheduler_num_cycles' 
``` 

Thanks in advance!

**Comment by sorryhyun** [comment]
sorry for that. It was silently removed but I'll restore it soon

**Comment by sorryhyun** [comment]
resolved in d5a1975
